src/tests/ftest/object/create_many_dkeys.yaml

```diff
@@ -4,13 +4,13 @@ hosts:
   test_servers:
     - server-A
     - server-B
-timeout: 8000
+timeout: 3000
 server_config:
   name: daos_server
 pool:
   mode: 511
   name: daos_server
-  scm_size: 2000000000
+  scm_size: 8G
```

Comment:
Is this change updated under master? If not, I am not sure whether you can merge it with 2.0? I believe the changes should work under master before propagating the changes to 2.0.

Reply:
This already landed on master.